docs/user-guide.md

```diff
@@ -1154,31 +1157,32 @@ Found p6880880_122010_Linux-x86-64.zip : OPatch Utility
 
 ## Prerequisite configuration
 
-Before you run the tool you need to create JSON formatted configuration files
-for the data mount devices and the ASM disk group.
+Before you run the tool you need to create JSON formatted configurations
+for the data mount devices and the ASM disk group. They can be stored in files or passed via CLI parameters.
 
 The [host provisinoing tool](host-provisioning.md) can configure newly-provisioned
 BMS hosts to run the toolkit installer, including authentication, Internet access,
 and local mountpoints.
 
-### Data mount configuration file
+### Data mounts configuration
 
-In the data mount configuration file, you specify disk device attributes for:
+In the data mounts configuration, you specify disk device attributes for:
 
-- Oracle software installation, which is usually mounted at /u01
-- Oracle diagnostic destination, which is usually mounted at /u02
+- Oracle software installation, which is usually mounted at `/u01`
+- Oracle diagnostic destination, which is usually mounted at `/u02`
 
-In the configuration file, specify the block devices (actual devices, not
+Specify the block devices (actual devices, not
 partitions), the mount point names, the file system types, and the mount options
-in valid JSON format.
+in a valid JSON format.
```

Comment:
I think the original here is correct: "in valid JSON format.".  (it's an uncountable noun)